test/test.js

```diff
@@ -37,4 +37,8 @@ describe('postcss-at2x', () => {
   it('should process image and add background size', (done) => {
     runTest('read-background-size.css', {detectImageSize: true}, done);
   });
+
+  it('should resolve image path with custom function', (done) => {
+    runTest('resolve-image-path.css', {resolveImagePath: (value) => {return path.resolve(process.cwd(), value);}}, done);
+  });
```

Comment:
Perhaps we could check the arguments passed to the callback are correct? Passing a spy as the option with `jest.fn()` would do it.

https://facebook.github.io/jest/docs/mock-function-api.html